build.gradle

```diff
@@ -130,6 +130,24 @@ subprojects {
           }
         }
 
+        configureThriftCompilation = {
```

Comment:
No need to do this here. Just move the contents to the thrift's build.gradle. We only did that for protobuf since it was used from multiple places.